src/site/apt/examples/set-compiler-source-and-target.apt.vm

```diff
@@ -28,21 +28,25 @@
 
 Setting the <<<-source>>> and <<<-target>>> of the Java Compiler
 
+  (If you're are using version <<<3.13.0>>> or newer of the Compiler Plugin, use the recommended
+  {{{../examples/set-compiler-release.html}<<<maven.compiler.release>>> property}}.)
+
```

Comment:
Note: this section may need to be updated later, after the work on `<sources>` element has been completed. The recommended approach would then be to specify the release together with the declaration of the sources.

No change needed now, it is just a reminder for revisiting later.